analyzer.js

```diff
@@ -123,8 +124,32 @@ const setFileRef = (allImports, file, pathway) => {
  */
 const parseFileASTTree = (file, allImports) => {
   const ast = parse(readFileSync(file, "utf-8"), babelParserOptions);
-  babelTraverse(ast, {
-    // Used for all ES6 import statements
+  traverse.default(ast, {
+    Import: (path) => {
+      if (path && path.node) {
+        setFileRef(allImports, file, path.node.source.value);
+      }
+    },
+    ImportDefaultSpecifier: (path) => {
+      if (path && path.node) {
+        setFileRef(allImports, file, path.node.source.value);
```

Comment:
Unfortunately, this commit introduced a few bugs which are getting masked with the try-catch. The functionality still works since we don't need to parse the import specifiers and attributes. This would be correct in a future PR.